src/containers/Node/NodeStructure/NodeStructure.tsx

```diff
@@ -47,23 +47,11 @@ function NodeStructure({nodeId, className}: NodeStructureProps) {
     ).query;
 
     const scrollContainerRef = useRef<HTMLDivElement>(null);
-    const scrollContainer = scrollContainerRef.current;
 
     const isReady = useRef(false);
 
     const scrolled = useRef(false);
 
-    useEffect(() => {
-        return () => {
-            if (scrollContainer) {
-                scrollContainer.scrollTo({
-                    behavior: 'smooth',
-                    top: 0,
-                });
-            }
-        };
-    }, [scrollContainer]);
-
```

Comment:
It occurred, this hook was never applied. Deleted it, as I don't understand, why it may be needed